setup.py

```diff
@@ -56,11 +56,14 @@
         # needed to process accounting data
         'numpy >= 1.14.0',
         'pandas >= 0.23.0',
```

Comment:
drop this line if you already have it below?

Reply:
right! :+1: